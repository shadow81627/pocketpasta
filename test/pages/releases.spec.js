import Component from '@/pages/releases.vue';
import { createLocalVue, shallowMount } from '@vue/test-utils';
import Vuetify from 'vuetify';
import BootstrapVuePlugin from 'bootstrap-vue';

const localVue = createLocalVue();

localVue.use(Vuetify);
localVue.use(BootstrapVuePlugin);

const factory = () =>
  shallowMount(Component, {
    localVue,
  });

describe('releases page', () => {
  test('mounts properly', () => {
    const wrapper = factory();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('renders properly', () => {
    const wrapper = factory();
    // wrapper.setMethods({ content: () => '# aBC' });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
