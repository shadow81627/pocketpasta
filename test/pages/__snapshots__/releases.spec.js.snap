// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`releases page renders properly 1`] = `"<div class=\\"container bg-light\\"><div class=\\"markdown-body\\">CHANGELOG.md</div></div>"`;
